README.md

```diff
@@ -555,4 +523,36 @@ HTTP status response: 500
     }
   ]
 }
-```
\ No newline at end of file
+```
+
+## For PEX-MS developers
+
+This project has been created using:
+* `yarn` version 1.22.11
+* `node` version 14.17.5
+* `docker-compose` version 1.26.2
+
+### Running
+1. Without docker:
```

Comment:
without docker you need a mongodb instance installed

Reply:
Fixed!